nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/main/java/org/apache/nifi/processors/kudu/PutKudu.java

```diff
@@ -165,7 +165,7 @@ public class PutKudu extends AbstractKuduProcessor {
         .name("Operation RecordPath")
         .displayName("Operation RecordPath")
         .description("If specified, this property denotes a RecordPath that will be evaluated against each incoming Record in order to determine the Kudu Operation Type. When evaluated, the " +
-            "RecordPath must evaluate to one of hte valid Kudu Operation Types, or the incoming FlowFile will be routed to failure. If this property is specified, the <Kudu Operation Type> property" +
+            "RecordPath must evaluate to one of the valid Kudu Operation Types, or the incoming FlowFile will be routed to failure. If this property is specified, the <Kudu Operation Type> property" +
```

Comment:
In my view it would worth mentioning in the parameter description or even in Additional Details that the Debezium operations (r, i,d,u) are supported and mapped to the PutKudu "standard" operations (INSERT, UPDATE etc.) otherwise users won't know about this improvement.